pallets/stateful-storage/src/lib.rs

```diff
@@ -120,7 +120,7 @@ pub mod pallet {
 
 		/// The maximum number of pages in a Paginated storage model
 		#[pallet::constant]
-		type MaxPaginatedPageId: Get<u32>;
+		type MaxPaginatedPageId: Get<u16>;
```

Comment:
@aramikm Can you validate that this is ok. Everything appears to be u16 for this lower down.

Reply:
This should have been u16. The issue I've seen before was that `Get<u16>` was not implementing all the traits that `Get<u32>` was, so either we had to use Get<u32> or implement those leftover traits that Parity didn't implement. This was a while ago so they might have added those traits to Get<u16> if you don't see any errors